parsec/mca/device/device.c

```diff
@@ -479,17 +483,18 @@ void parsec_devices_print_statistics(parsec_context_t *parsec_context, uint64_t
         total_required_in  += required_in[i];
         total_required_out += required_out[i];
         total_d2d          += transferred_d2d[i];
+        total_evicted      += nb_evictions[i];
     }
 
     /* Print statistics */
     gtotal = (float)total_tasks;
     double percent_in, percent_out, percent_d2d;
 
-    printf("+----------------------------------------------------------------------------------------------------------------------------+\n");
-    printf("|         |                    |                       Data In                              |         Data Out               |\n");
-    printf("|Rank %3d |  # KERNEL |    %%   |  Required  |   Transfered H2D(%%)   |   Transfered D2D(%%)   |  Required  |   Transfered(%%)   |\n",
+    printf("+-----------------------------------------------------------------------------------------------------------------------------------------------+\n");
+    printf("|         |                    |                       Data In                              |         Data Out               |                  |\n");
+    printf("|Rank %3d |  # KERNEL |    %%   |  Required  |   Transfered H2D(%%)   |   Transfered D2D(%%)   |  Required  |   Transfered(%%)   |   nb evictions   |\n",
```

Comment:
A bit more consistent:
```suggestion
    printf("|Rank %3d |  # KERNEL |    %%   |  Required  |   Transfered H2D(%%)   |   Transfered D2D(%%)   |  Required  |   Transfered(%%)   |   Evictions   |\n",
```